configs/defaults/gcnv.toml

```diff
@@ -0,0 +1,7 @@
+[workflow]
+intervals_path = 'gs://cpg-common-main/references/hg38/v0/exome_calling_regions.v1.interval_list'
```

Comment:
This file is already part of the core reference set, so you can pull from there

```toml

[references.broad]
dragmap_prefix = "gs://cpg-common-main/references/hg38/v0/dragen_reference"
...
exome_calling_interval_lists = "gs://cpg-common-main/references/hg38/v0/exome_calling_regions.v1.interval_list"
```

Reply:
I'll track down the better way of getting these, but you can use the `config` mode of analysis-runner to generate the combined config your current run will create, e.g. 
```
analysis-runner config --dataset broad-rgp -o here.toml --access-level test --config configs/genome.toml --config configs/gatk-sv-broad-rgp-test.toml
```

This will generate the exact run config you'd create by submitting a workflow run instruction + those two manually provided configs

Reply:
Um, yes, that's the `gs:` URL I copied in there?

Unless you're suggesting to just use `gs://cpg-common-main/references/…/preprocessed_intervals.interval_list` rather than generating it in the `PrepareIntervals` stage?

Reply:
Oh, you mean access it via the existing config key using something like `reference_path('broad/exome_calling_interval_lists')`.

Probably. One would have to think about what would change if we ever switched target intervals…

Reply:
Yeah sorry, I wasn't clear about what I meant here. This exact file is already part of the standard references so it can be pulled from there. It's a really minor nit so I wouldn't worry about it